packages/template-block/package.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "template-block",
+    "private": true,
+    "author": "Frontify Developers <developers@frontify.com>",
+    "version": "1.0.0",
+    "main": "dist/index.js",
+    "scripts": {
+        "serve": "frontify-cli serve --entryPath src/index.ts",
+        "deploy": "frontify-cli deploy --entryPath src/index.ts",
+        "lint": "eslint .",
+        "lint:fix": "eslint --fix .",
+        "prettier": "prettier --check .",
+        "prettier:fix": "prettier --write .",
+        "typecheck": "tsc --noEmit"
+    },
+    "devDependencies": {
+        "@babel/core": "^7.22.15",
+        "@frontify/eslint-config-react": "0.16.1",
+        "@frontify/frontify-cli": "^5.3.17",
+        "@types/react": "^18.2.21",
+        "@types/react-dom": "^18.2.7",
+        "autoprefixer": "^10.4.15",
+        "cypress": "^13.1.0",
+        "cypress-real-events": "^1.10.1",
+        "eslint": "^8.48.0",
+        "eslint-plugin-notice": "^0.9.10",
```

Comment:
```suggestion
        "eslint": "^8.48.0",
        "eslint-plugin-jsx-a11y": "^6.7.1",
        "eslint-plugin-notice": "^0.9.10",
```

Reply:
@ruslan-sazonov you need to rerun `pnpm i` and then commit the lockfile to add it properly